Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,501 @@
+Dr.\ Patru would certainly be a great resource for this kind of help.
+
+The basic coding abilites include writing a MATLAB script to analyze and plot data. MATLAB is easy to learn and is very powerful. It is a very common choice for data analysis for mechancial engineering
+projects where the core skill set is not computer science. This is not hard for anyone who has coded before.
+
+The machining will be the most difficult to learn if there are no MEs or METs that have taken the proper courses. Machining can be extremely dangerous and cannot be done
+without the proper safety training. If no one is trained on the machines one can ask a machinist to make simple parts or submit a job at the Earl Brickman Machine Tools and Manufacturing Lab.
+They have professional engineers there that can make nearly anything one could need. But their timeline slips often, so leave an appropriate amount of time for them to make the part
+so it does not interfere with the projects timeline.
+
+\subsection{Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+Fortunately, most of the materials for the test stand have been purchased. This eliminates much of the system cost. The most costly part of the project
+will be the propulsion system (the regulator, solenoid, nozzle, piping, and interfaces). A preliminary budget that includes margin for cost overruns is \$600.
+The total cost was approximately \$450. That would be the last part of the system needed before testing can occur. Accounting for unforseen cost then rounding upwards,
+the consquentical estimated cost is put at \$600 for the entirty of the project. A bill of materials is displayed in Appendix B. The table, displayed below, outlines
+the most expensive parts that the team will need to purchase for the propulsion system, these parts are worst-case scenario in cost and measures will be taken to
+try to alleviate these costs.
+
+\begin{table}
+  \centering
+  \begin{tabularx}{\columnwidth}{@{}cXc@{}cXc@{}}
+
+    \multicolumn{3}{ c }{\textbf{Propulsion System Part List}} \\ \toprule
+
+    Part & Cost & McMaster Part \# \\ \midrule
+
+    Tank & \$ 131.35 & 7822A11 \\
+
+    Regulator & \$ 182.50 & 49305K21 \\
+
+    Solenoid & \$ 100.49 & 4711K511 \\
+
+    \bottomrule
+  \end{tabularx}
+\label{tab:propulsion-part-list}
+\end{table}
+
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+Facculty support will be necessary. The SPEX propulsion team has consulted faculty about structure and flow system concerns and made the suggested changes. However, the propulsion system
+has not been tested. It is likely that faculty assistance will be necessary to get the propulsion to a functional stage. This would most likely inlcude the design of an optimally expanding
```

Comment:
~the propulsion~ the propulsion system.

propulsion is not a noun in this case.